helix-rest/src/main/java/org/apache/helix/rest/metadatastore/accessor/ZkRoutingDataWriter.java

```diff
@@ -332,8 +349,9 @@ private String constructUrlSuffix(String... urlParams) {
     return String.join("", allUrlParameters);
   }
 
-  private boolean forwardRequestToLeader(String urlSuffix, HttpConstants.RestVerbs request_method,
-      int expectedResponseCode) throws IllegalArgumentException {
+  private boolean buildAndSendRequestToLeader(String urlSuffix,
+      HttpConstants.RestVerbs request_method, int expectedResponseCode)
```

Comment:
shouldWeUseCamelCaseFor request_method?

Reply:
Ah sorry, python habits coming through. Fixing it.